### ISSUE TITLE
Global items

### ISSUE THREAD

**Issue by vitkarpov** [javascript-playground/backbone-require-example#3]
Is it really good idea to use global var `items`? We operate it in the cart collection`s add event.

We should customize Backbone.sync to handle it, shouldn`t we?

**Comment by jackfranklin** [comment]
You're right - there's no need for `items` to be global. I would probably move it inline and just do:

```
new CartCollextion([
  {item1...},
  {item2...}
])
```

Or similar. Of course, if you have a server, it would make more sense to set up the collection to fetch from the server.

**Comment by jahnkajavaee** [comment]
Thanks for your information

<a href="http://javascripttrainingcourses.com/JS/BackboneJS-Training" title="Backbone.JS Training in Chennai">Backbone.JS Training in Chennai</a>
